package.json

```diff
@@ -111,7 +111,7 @@
     "saxes": "^5.0.1",
     "tmp": "^0.2.0",
     "unzipper": "^0.10.11",
-    "uuid": "^8.3.0"
+    "uuid": "^9.0.0"
```

Comment:
I remember last year uuid released v9 to address vulnerability issue so it's always good to keep up to date with it